projects/kit/components/pulse/pulse.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    EventEmitter,
+    HostBinding,
+    inject,
+    Input,
+    Output,
+} from '@angular/core';
+import {
+    TUI_ANIMATIONS_SPEED,
+    tuiFadeIn,
+    tuiScaleIn,
+    tuiToAnimationOptions,
+} from '@taiga-ui/core';
+
+@Component({
+    standalone: true,
+    selector: 'tui-pulse',
+    template: '',
+    styleUrls: ['./pulse.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    animations: [tuiFadeIn, tuiScaleIn],
+})
+export class TuiPulseComponent {
+    @Input()
+    public playing = true;
+
+    @Output()
+    public readonly playingChange = new EventEmitter<boolean>();
```

Comment:
Unnecessary, it does not change state internally.

Reply:
fixed